migrations/2023-06-20-191145_add_listingtype_sorttype_3_6_9_months_enums/down.sql

```diff
@@ -0,0 +1,14 @@
+-- update the default sort type
+update local_user set default_sort_type = 'TopDay' where default_sort_type in ('TopThreeMonths', 'TopSixMonths', 'TopNineMonths');
+
+-- rename the old enum
+alter type sort_type_enum rename to sort_type_enum__;
+-- create the new enum
+CREATE TYPE sort_type_enum AS ENUM ('Active', 'Hot', 'New', 'Old', 'TopDay', 'TopWeek', 'TopMonth', 'TopYear', 'TopAll', 'MostComments', 'NewComments');
```

Comment:
this should include the TopHour, TopSixHour and TopTwelveHour enums

Reply:
You're right, I was not sure when I made the PR

Reply:
usually there's no take backsies with migrations, but I think in this case it's a down migration and is probably fine just opening a new PR with the hotfix of this file.

Reply:
Opened a PR #3370